src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandler.php

```diff
@@ -160,7 +160,7 @@ public function write($sessionId, $data)
                 $mergeStmt->bindParam(':data', $encoded, \PDO::PARAM_STR);
                 $mergeStmt->bindValue(':time', time(), \PDO::PARAM_INT);
 
-                //Oracle has a bug that will intermitently happen if you
+                //Oracle has a bug that will intermittently happen if you
                 //have only 1 bind on a CLOB field for 2 different statements
                 //(INSERT and UPDATE in this case)
```

Comment:
Can you make the whitespace change here that @javiereguiluz suggested in https://github.com/symfony/symfony/pull/16098#discussion_r41088015?